server/s3_management_api_handler.cc

```diff
@@ -30,10 +31,17 @@ void S3ManagementAPIHandler::create_action() {
     case S3OperationCode::none:
       // Perform operation on Service.
       switch (request->http_verb()) {
-        case S3HttpVerb::DELETE:
+        case S3HttpVerb::DELETE: {
           action = std::make_shared<S3AccountDeleteMetadataAction>(request);
           s3_log(S3_LOG_DEBUG, request_id, "S3AccountDeleteMetadataAction");
-          break;
+        } break;
```

Comment:
break above } ?

Reply:
why do we need {} here? it's not rquired and I dont think we use braces in rest of code inside `switch` statements...

Reply:
to resolve issue of not handling post put enumaration values,i had to add this {},which resolved the issue.